killrweather-dashboard/app/com/datastax/killrweather/KafkaPublisherActor.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.killrweather
+
+import kafka.producer.ProducerConfig
+import kafka.serializer.StringEncoder
+
+import com.datastax.spark.connector.embedded._
+
+class KafkaPublisherActor(val producerConfig: ProducerConfig) extends KafkaProducerActor[String,String] {
+
+  def this(hosts: Set[String], batchSize: Int) = this(KafkaProducer.createConfig(
+    hosts, batchSize, "async", classOf[StringEncoder].getName))
+
+}
```

Comment:
ah, what a pain. let's leave this here for now then and I'll circle back later and see if I can fix the issue.